beer-garden/beer-garden
# Issue/1228 Adds api for interacting with command block list

Adds api to add and remove  commands from the command publishing block list, and get the commands that are in the block list

## Testing Instructions
Make the api calls through terminal or swagger. If using authentication there needs to be an access token sent with api call.

1. Use the api to make a post to add command to block list. Example input: 
```
[
  {
    "system": "string",
    "command": "string",
    "namespace": "string"
  }
]
```
2. Use api to get list of commands in the command publishing block list, which should contain the command(s) added from the post.
3.  Use an id for one of the commands in the list that was returned to make an api call to delete a command, and repeat step 2 to verify command was removed.

If command is from child garden the command should be added or removed from parent and child if api call to post or delete are made from the parent. Also the entries on parent and child should have matching ids. To verify commands added to the child a get needs to be performed on the child.
